numalogic/synthetic/anomalies.py

```diff
@@ -31,11 +39,19 @@ def __init__(self, ref_df: pd.DataFrame, anomaly_type="global", anomaly_ratio=0.
             - "collective": Outliers present simultaneously in two or more time series
         @param anomaly_ratio: Ratio of anomalous data points to inject wrt
             to number of samples
+        @param anomaly_sign: Positive or Negative anomaly to be injected
+            Possible values include:
+                - "positive": positive outliers injected only
+                - "negative": negative outliers injected only
+        @param mu: Distributions mean of the Gaussian Noise injected
+        @param sigma: Distributions std of the Gaussian Noise injected
```

Comment:
Can you normalize the docstring format? I think we use a different format elsewhere in the code

Reply:
Yes